src/store/ShareStore.js

```diff
@@ -1,7 +1,10 @@
-import html2canvas from 'html2canvas';
 import { observable, action, reaction, computed, when } from 'mobx';
 import MenuStore from './MenuStore';
 import { downloadFileInBrowser, findAncestor } from './utils';
+import { loadScript } from '../utils';
+
+const html2canvasCDN = 'https://html2canvas.hertzen.com/dist/html2canvas.min.js';
+setTimeout(() => loadScript(html2canvasCDN), 2000);
```

Comment:
I think we should load the scrip on demand, the first time that user clicks download as PNG.

Reply:
[I initially did that](https://github.com/binary-com/SmartCharts/pull/317/commits/74d16827785031cacdc8ada9be71061a6c01d32c). But then after discussing with mohammed we figured we should have it load after the chart loads, otherwise there will be a slight pause to download a 36Kb file.